RELEASE/scripts/autoscend/auto_bedtime.ash

```diff
@@ -662,9 +662,9 @@ boolean doBedtime()
 
 	if((friars_available()) && (!get_property("friarsBlessingReceived").to_boolean()))
 	{
-		if(in_pokefam() || in_darkGyffte())
+		if(is_boris() || is_jarlsberg() || is_pete() || isActuallyEd() || in_darkGyffte() || in_lta() || in_pokefam())
```

Comment:
Use `pathHasFamiliar()` instead to reduce the need for this?

Reply:
Thanks!  I put the "has familiar" logic first and the food one as a backup